### PR TITLE
refactor/1022 - Reintroduce Flughafen München (EDDM)

### LINKED ISSUE #1022: Airport Revival: Flughafen München
Munich Airport (IATA: MUC, ICAO: EDDM), German: Flughafen München, is a major international airport near Munich, the capital of Bavaria. It is the second-busiest airport in Germany in terms of passenger traffic after Frankfurt Airport, and the seventh-busiest airport in Europe, handling 44.6 million passengers in 2017. It is the world's 15th-busiest airport in terms of international passenger traffic, and was the 34th-busiest airport worldwide in 2015. As of March 2018, the airport features flights to 266 destinations, making it the airport with the fifth-most destinations worldwide. Munich Airport serves as a hub for Lufthansa including Lufthansa Regional and its Star Alliance partners. It was named Europe's best airport in 2018 and ranks 6th on the worldwide ranking.

### PR DESCRIPTION
**Pull request by Marcel510**
Resolves #1022
Brings back Munich airport (EDDM)

- [x] basic values
- [x] terrain file
- [x] airspace
- [x] RNAV SIDs
- [x] non-RNAV SIDs
- [x] RNAV STARs
- [x] non-RNAV STARs
- [x] spawnpatterns
  - [x] airline updates
  - [x] airways
- [x] restricted airspace
- [x] videomap
- [x] airport guide
- [x] changelog

### CODE REVIEW

**Comment by guyndumais** [comment]
@Marcel510 EDDM spawn patterns is not a gimme! Need any help on this?

File: assets/airports/airportLoadList.js [hunk 1]
```diff
@@ -87,12 +87,12 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
...
-        //     icao: 'eddm',
-        //     level: 'hard',
-        //     name: 'Franz Josef Strauß International Airport',
-        //     premium: false
-        // },
+        {
+            icao: 'eddm',
+            level: 'hard',
+            name: 'Flughafen München',
+            wip: false
```
Comment (Fechulo) [inline assets/airports/airportLoadList.js]: wip property isn't used anymore, replace this with premium

File: documentation/airport-guides/eddm.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+* **Total passengers**: *46,253,623*
+* **Aircraft movements**: *413,469*
+* **Cargo**: *368,377 tonnes*
+
+### About the facility
+There are four Area Control Centers in Germany: Bremen, Langen, Karlsruhe and München, operated by the Deutsche Flugsicherung (German Air Traffic Control). The terminal control service for EDDM is provided by the München ACC which is located at the airport. München ACC also provides terminal control services for Oberpfaffenhofen Airport (EDMO) and Augsburg Airport (EDMA) as well as air traffic services for en-route traffic within the München Flight Information Region (EDMM FIR).
+
+### Airspace
+The airspace encompasses a large area around Munich up to FL195. Big parts of the airspace are classified as Class C airspace (equivalent to American class B airspace). As long as one approach radar system (ASR) is in operation 3nm horizontal separation may be applied.
```
Comment (Fechulo) [inline documentation/airport-guides/eddm.md]: As long as one approach radar system (ASR) is in operation **_,_** 3nm horizontal separation may be applied. (Add comma)

File: documentation/airport-guides/eddm.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+### Runways
+EDDM consists of two parallel concrete runways (08R/26L and 08L/26R). Both runways have equal length (4000m / 13123ft) and width (60m / 197ft) and are equipped with instrument landing systems in both directions. A third runway north-east of 08R/26L is planned but construction has not yet begun due to protests of residents.
```
Comment (Fechulo) [inline documentation/airport-guides/eddm.md]: Both runways **are equal in** length and width
Protests **by** residents

File: documentation/airport-guides/eddm.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+#### Runway usage
+Runways 26L and 26R are preferentially used as long as the tailwind component does not exceed 5 knots. The runways are approved for independent departure and independent parallel approach operations. Radar separation of at least 3nm and/or 1000ft vertical separation has to be ensured until both aircraft are stabilized on the localizer.
```
Comment (Fechulo) [inline documentation/airport-guides/eddm.md]: Separation has to be **maintained** until aircraft are **established** on the localiser

File: documentation/airport-guides/eddm.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+| **VAVOR**    | **3S***/3W² |    3N   |    3Q   | **3E*** |
+
+\* Preferred departure route  
+¹ Non-jet aircraft  
+² For aircraft unable to comply with restrictions of the "S"-suffix SID
+
+
+#### Arrivals
+
+EDDM offers a variety of standard terminal arrival routes that lead to one of the four arrival gates: ROKIL, NAPSA, BETOS, LANDU. These waypoints are clearance limits. If no further clearance is issued by the controller aircraft have to enter the published holding. After the arrival gate transitions (e.g. ROKIL26), specific terminal waypoints (DMxxx) or radar vectors are used to guide the aircraft to the ILS.
```
Comment (Fechulo) [inline documentation/airport-guides/eddm.md]: enter the published **hold**

**Comment by Fechulo** [comment]
Amazingly, I couldn't find anything in the airport file, fair play! Practical review still pending

**Comment by Marcel510** [comment]
@Fechulo 
1. Yep, that's also I problem I noticed when testing the airport. In reality that's probably not a big deal since the handoff by the previous station will be issued 10nm prior to the Sector border. But that's not the way it's done in openScope, so the obvious solution would be to enlarge the airspace. I personally don't really like this solution as I'd prefer to keep the airspace as true to the original as possible. 
Therefore I'd also prefer the latter. Do you want me to add a small note to the airport guide about this deviation from reality?

2. Looks a bit weird but that's the way it is in reality. Probably due to the sectorization of the EDMM FIR's airspace.
Screenshot_358

**Comment by Fechulo** [comment]
> Do you want me to add a small note to the airport guide about this deviation from reality?

You read my mind 😉

**Pull request merged**